core/src/main/java/io/questdb/cairo/CairoException.java

```diff
@@ -41,16 +42,20 @@ public class CairoException extends RuntimeException implements Sinkable, Flywei
     private boolean cacheable;
     private boolean interruption;
 
-    public static CairoException duplicateColumn(CharSequence colName) {
-        return duplicateColumn(colName, null);
+    public static CairoException duplicateColumn(CharSequence columnName) {
+        return duplicateColumn(columnName, null);
     }
 
-    public static CairoException duplicateColumn(CharSequence colName, CharSequence colAlias) {
-        CairoException exception = instance(METADATA_VALIDATION).put("Duplicate column [name=").put(colName);
-        if (colAlias != null) {
-            exception.put(", alias=").put(colAlias);
+    public static CairoException duplicateColumn(CharSequence columnName, CharSequence columnAlias) {
+        CairoException exception = metadataValidation("Duplicate column", columnName);
+        if (columnAlias != null) {
+            exception.put(", [alias=").put(columnAlias).put(']');
         }
-        return exception.put(']');
+        return exception;
+    }
+
+    public static CairoException metadataValidation(@NotNull CharSequence msg, @NotNull CharSequence columnName) {
```

Comment:
this is an unfortunate name perhaps, can we call this `invalidMetadata` - to be factual